panels/models/attraction.py

```diff
@@ -666,7 +666,7 @@ def start_time_label(self):
     def checkin_time(self):
         advance_checkin = self.attraction.advance_checkin
         if advance_checkin < 0:
-            return self.start_time
+            return self.end_time
         else:
             return self.start_time - timedelta(seconds=advance_checkin)
```

Comment:
We don't have to write up a separate `elif` clause for option 0 ("When the event starts") because start_time minus 0 = start_time.